_includes/example/in-page-config.html

```diff
@@ -1,3 +1,4 @@
+<meta property="laterpay:connector:article_id" content="custom_article_id_for_in_page_config_example_page">
```

Comment:
Shouldn’t we use some kind of ID here? Like `ID1234567890`? Currently it’s not very clear what value would be valid in here.